src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import cx from 'classnames'
+import isString from 'lodash/isString'
```

Comment:
The standard pattern in this project has been to include lodash as `_` as the very first import within the file. We use the `babel-plugin-lodash` plugin which handles cherry-picking only the functions you use (see https://github.com/lodash/babel-plugin-lodash#example)